src/services/htmlStorage.service.ts

```diff
@@ -5,7 +5,7 @@ import { ConstantsService } from 'jslib-common/services';
 export class HtmlStorageService implements StorageService {
     private localStorageKeys = new Set(['appId', 'anonymousAppId', 'rememberedEmail', 'passwordGenerationOptions',
         ConstantsService.disableFaviconKey, 'rememberEmail', 'enableGravatars', 'enableFullWidth',
-        ConstantsService.localeKey, ConstantsService.autoConfirmFingerprints,
+        ConstantsService.themeKey, ConstantsService.localeKey, ConstantsService.autoConfirmFingerprints,
```

Comment:
I think you can revert this change now that you're not making any other changes in this file.